entries/c/clubhouse.com.json

```
@@ -0,0 +1,11 @@
+{
+  "Clubhouse": {
+    "domain": "clubhouse.com",
+    "contact": {
+      "twitter": "Clubhouse"
```

```suggestion
      "twitter": "Clubhouse",
      "form": "https://5nhc6u4b3x3.typeform.com/to/Asez2lqx"
```

I found this feature request form on [their support site](https://support.clubhouse.com/hc/articles/4405242707603).

---

Good find! I could only find that 'submit a request' and all of those topics didn't really match suggesting/requesting 2FA.

---

Happy to help!